app/src/pages/inside/common/infoLine/share/shareTooltip/shareTooltip.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { defineMessages } from 'react-intl';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './shareTooltip.scss';
+
+const cx = classNames.bind(styles);
+
+const messages = defineMessages({
+  sharedByFilter: {
+    id: 'ShareTooltip.sharedByFilter',
+    defaultMessage: 'Filter is shared by {owner}',
```

Comment:
This is the third place where I can see this text. Looks like it's time to extract icon message and tooltip to a common component